module/move/wca/src/ca/grammar/command.rs

```diff
@@ -4,8 +4,11 @@ pub( crate ) mod private
 
   use { Handler, Routine, Type };
 
-  use std::collections::HashMap;
+  use std::collections::{ BTreeMap, HashMap };
   use former::{ Former, StoragePreform };
+  use crate::ca::aggregator::private::Order;
```

Comment:
This seems to be not in line with the [convention](https://github.com/obox-systems/conventions/blob/master/code_style.md#importing-local-entities)